### DISCUSSION TITLE
During live video streaming seek bar not being displayed, only able to see Live

### DISCUSSION THREAD

**Discussion by senin142** [videojs/video.js#8728] [General]
Explain in detail the exact steps necessary to reproduce the issue.

    1.Load video with the intended source URL.
    2.liveui option as true,liveTracker: {trackingThreshold: 30s}.
    3.Video is on autoPlay.Play the live video and it the issue would replicate. No seek bar and only Live is displayed
    (Video is 24*7 continuous play of a news channel)

Results
Expected

While streaming live videos seek bar should be visible.
Actual

During live video streaming seek bar not being displayed, only able to see Live
Error output

N/A

Additional Information:
 "video.js": "^8.10.0",
 
 Additional Plugin:
 "videojs-contrib-dash": "^5.1.1",
 "videojs-contrib-quality-levels": "^4.1.0",
 "videojs-errors": "^6.2.0",
 "videojs-hls-quality-selector": "^2.0.0",
 "videojs-sprite-thumbnails": "^2.2.1",
   
 browsers :Chrome, Firefox

**Comment by campones** [comment]
no wonder nobody commented. 
well I m having the real issue which is I am playing a dash live stream and live icon is not showing up , and seeking bar is.
